### DISCUSSION TITLE
Is there a way to trigger other bots using my bot?

### DISCUSSION THREAD

**Discussion by Micahthebomb** [Rapptz/discord.py#6809] [Q&A]
I have a bot idea that revolves around the idea of my bot triggering other bots but I can't seem to find a way to make it work. Any suggestions?

**Comment by Rapptz** [comment]
The library doesn't allow bots to invoke other bots since that behaviour tends to be abusive more than not.

**Reply by Micahthebomb** [reply]
Are there any other libraries possibly in other languages that allow this?

**Reply by Rapptz** [reply]
If you want to forego using the commands extension you can just do command parsing manually and use `on_message` which doesn't care where the message comes from or you can special case the commands you care about within `on_message`.
